python-capdl-tool/capdl/util.py

```diff
@@ -239,8 +239,12 @@ def ipc_buffer_size(self):
         return 512
 
 
+# Support for ARMv6 has been removed from seL4 in early 2022. However, support
+# for "arm11" is kept here, because this name is used in the CapDL specification
+# for AARCH32 configurations. Updating this is a low priority task, because it
+# is a lot of work with not much gain (except cleaning up legacy), Also, keeping
+# the name there isn't causing any issues.
 CAPDL_SUPPORTED_ARCHITECTURES = {
-    # <name>:  [arch_obj_ctor, <alias_list>]
```

Comment:
Did you intend to remove this comment? It seems useful, but if you did then you might as well remove it from the first commit.

Reply:
Good catch, I will bring it back

Reply:
done